ga4gh/backend.py

```diff
@@ -413,6 +429,18 @@ def getVariantSet(self, id_):
         dataset = self._getDatasetFromCompoundId(id_)
         return self.runGetRequest(dataset.getVariantSetIdMap(), id_)
 
+    def getVariant(self, id_):
+        """
+        Returns a variant with the given id
+        """
+        compoundId = variants.CompoundVariantId(id_)
+        dataset = self._getDatasetFromCompoundId(compoundId.datasetId)
+        variantSet = _safeMapQuery(
```

Comment:
Why don't you like having a try/catch in datasets.py? Seems simple enough to me:
```python
def getVariantSet(variantSetId):
    if variantSetId not in self._variantSetIdMap:
         raise VariantSetNotFoundException(variantSetId)
    return self._variantSetIdMap[variantSetId]
```